nym-vpn-app/package.json

```diff
@@ -14,7 +14,7 @@
     "dev": "vite",
     "dev:app": "tauri dev",
     "dev:browser": "vite --mode dev-browser",
-    "check": "run-s lint tscheck fmt",
+    "check": "run-s lint tscheck fmt test",
```

Comment:
Please, remove test from the check script
we do not want it here as running unit tests is a heavy task
and that check command is to perform quick checks during dev iterations